test/testother.cpp

```diff
@@ -1551,14 +1551,15 @@ class TestOther : public TestFixture {
         ASSERT_EQUALS("", errout.str());
 
         check("class X {\n"
-              "    uint64_t i;\n"
+              "    void* i;\n"
               "};\n"
               "class Y : X {\n"
-              "    uint64_t j;\n"
+              "    void* j;\n"
+              "    void* k;\n"
               "};\n"
               "void f(X x, Y y) {\n"
               "}");
-        ASSERT_EQUALS("[test.cpp:7]: (performance) Function parameter 'y' should be passed by reference.\n", errout.str());
+        ASSERT_EQUALS("[test.cpp:8]: (performance) Function parameter 'y' should be passed by reference.\n", errout.str());
```

Comment:
yes inheritance is one thing it tested... I see no other test that uses uint64_t. It might be interesting to test that it will generate a warning in 32-bit platforms but not in 64-bit platforms. Same code but two different Settings::platformType should generate different results.